sklearn/preprocessing/_encoders.py

```diff
@@ -731,6 +765,13 @@ def inverse_transform(self, X):
 
         for i in range(n_features):
             labels = X[:, i].astype('int64', copy=False)
-            X_tr[:, i] = self.categories_[i][labels]
+            # set unknown values to None
+            if self.handle_unknown == 'use_encoded_value':
+                X_tr[:, i] = np.where(
+                    labels == self.unknown_value, None,
```

Comment:
We can set `unknown_labels = labels == self.unknown_value` so the mask does not need to be computed twice.